observer/probers/dns/dns_conf_test.go

```diff
@@ -167,6 +184,19 @@ func TestDNSConf_MakeProber(t *testing.T) {
 	}
 }
 
+func TestDNSConf_Instrument(t *testing.T) {
+	t.Run("instrument", func(t *testing.T) {
+		conf := DNSConf{}
+		colls := conf.Instrument()
+		for name := range colls {
+			switch name {
+			default:
+				t.Errorf("DNSConf.Instrument() returned unexpected Collector '%s'", name)
+			}
+		}
+	})
+}
+
```

Comment:
Since we don't need to implement `Instrument()` for the `DNSConf`, this test can go. 

```suggestion
```